games-roguelike/angband/angband-4.1.3.ebuild

```diff
@@ -16,6 +16,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="ncurses sdl +sound X"
```

Comment:
Now it seems weird to me. We have `REQUIRED_USE="sound? ( sdl )"`. Shouldn't it be `IUSE="ncurses +sdl +sound X"` ('+' for sdl)? Or remove '+' for sound?

Reply:
The `+` marks are usually used to indicate what upstream supports the defaults to be. I checked configure.ac and this one, by default, enabled x11 and ncurses, and disables sdl. Also it by default enables sound IF --enable-sdl is used.

What I would do, is `+ncurses sdl sound +X`

Reply:
Oh, thanks.